packages/cli/src/types.d.ts

```diff
@@ -258,3 +258,8 @@ declare module 'webpack-node-externals' {
     }
   }
 }
+
+/** TODO: Remove when rjsf actually ship types that match the package exports. */
+declare module '@rjsf/material-ui/v4' {
```

Comment:
Really want to move this type somewhere else if we can, hate that it's in the CLI rather than shipped with the scaffolder instead.